watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -277,37 +277,44 @@ def build(self):
         # components
         self.H2O = Solvent()
 
-        # blank sets
-        self.cation_set = Set()
-        self.anion_set = Set()
-        self.solute_set = Set()
-        self.ion_set = Set()
```

Comment:
@lbibl Can you remind me why `ion_set` was excluded? I think it's because `ion_set` is already accounted for via IDAES, but I'm forgetting. I only ask because I anticipate that resolving the failure related to the NF DSPMDE model will require both an ion_set and solute_set. It could be easily handled without any modification here, but it would be convenient to just have it defined here. 

And if it is still the case the `ion_set` is already generated via IDAES, then I might suggest adding a comment here saying so as a reminder.

Reply:
Good catch.  The reason is that in IDAES the `Anion` and `Cation` class inherit `Ion` class.  It is current coded in the way that when you put an attribute in `Anion` (or `Cation`), you have to take it out from the `Ion` class, i.e., you cannot have a component in both `Anion` and `Ion`.   Because of this, IDAES document that "Users are encouraged to use the Anion and Cation classes instead of the generic Ion class, as these validate that sign of the charge configuration argument." (https://idaes-pse.readthedocs.io/en/stable/reference_guides/core/comp.html#ion-class).  This design could be improved from IDAES but this is what is currently.   I still added an ion_set so we have something to call `anion_set|cation_set`.  This is the only way to do this as far as I see.

Reply:
`solute_set` in this PR is equivalent to `ion_set|solute_set` previously.  The previous `solute_set` is equivalent to `neutral_set` in this PR.